_shared/project/requirements/clean.in

```diff
@@ -0,0 +1,6 @@
+pip-tools
+pip-sync-faster
+coverage[toml]
+{% if include_exists("requirements/clean.in") %}
+    {{- include("requirements/clean.in") -}}
+{% endif %}
```

Comment:
This is kind of annoying but I had to add a new `clean` environment to tox (see below) and so for apps we need `clean.{in,txt}` requirements files as well